### PR TITLE
[Docs] [#14813] Document the 'get_replication_status' yb-admin command

### PR DESCRIPTION
**Pull request by joe-maley**

### CODE REVIEW

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 1]
```diff
@@ -137,6 +137,22 @@ Replication lag is computed at the tablet level as follows:
 
 To obtain information about the overall maximum lag, you should check `/metrics` or `/prometheus-metrics` for `async_replication_sent_lag_micros` or `async_replication_committed_lag_micros` and take the maximum of these values across each source's T-Server. For information on how to set up the node exporter and Prometheus manually, see [Prometheus integration](../../../explore/observability/prometheus-integration/linux/).
 
+### Replication status
+
+The `yb-admin` tool can be used to return the current replication status. The `get_replication_status` command will return the replication status for all *consumer-side* replication streams. An empty "errors" field should be interpretted as a healthy replication stream.
+
+```
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:144]: code block is sh

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 2]
```diff
@@ -137,6 +137,22 @@ Replication lag is computed at the tablet level as follows:
 
 To obtain information about the overall maximum lag, you should check `/metrics` or `/prometheus-metrics` for `async_replication_sent_lag_micros` or `async_replication_committed_lag_micros` and take the maximum of these values across each source's T-Server. For information on how to set up the node exporter and Prometheus manually, see [Prometheus integration](../../../explore/observability/prometheus-integration/linux/).
 
+### Replication status
+
+The `yb-admin` tool can be used to return the current replication status. The `get_replication_status` command will return the replication status for all *consumer-side* replication streams. An empty "errors" field should be interpretted as a healthy replication stream.
```
[the remark this answers is not in the document]
Comment (joe-maley) [reply docs/content/preview/deploy/multi-dc/async-replication.md]: Done, thanks.

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 1 above)
Comment (joe-maley) [reply docs/content/preview/deploy/multi-dc/async-replication.md]: Done, thanks.

**Comment by joe-maley** [comment]
@ddhodge I have also added changes to the yb-admin reference page. Please review these new changes. Thank you.

File: docs/content/preview/admin/yb-admin.md [hunk 3]
```diff
@@ -1840,6 +1840,38 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
 
+#### get_replication_status
+
+Returns the replication status of all consumer streams. If *producer_universe_uuid* is provided, this will only return streams that belong to an associated universe key.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> get_replication_status [ <producer_universe_uuid> ]
```
Comment (polarweasel) [inline docs/content/preview/admin/yb-admin.md:1851]: Just for consistency...
```suggestion
./bin/yb-admin \
    -master_addresses <master-addresses> \
    get_replication_status [ <producer_universe_uuid> ]
```

File: docs/content/preview/admin/yb-admin.md [hunk 4]
```diff
@@ -1840,6 +1840,38 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
...
+
+**Example**
+
+```sh
+./bin/yb-admin \
+    -master_addresses 172.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+    get_replication_status e260b8b6-e89f-4505-bb8e-b31f74aa29f3
+```
+
+```output
```
Comment (polarweasel) [inline docs/content/preview/admin/yb-admin.md:1864]: Is this output correct? (As in, not actually valid JSON, just JSON-like?)

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 5]
```diff
@@ -137,6 +137,22 @@ Replication lag is computed at the tablet level as follows:
...
+./yb-admin -master_addresses 127.0.0.1:7000,127.0.0.2:7000,127.0.0.3:7000 get_replication_status
+statuses {
+  table_id: "03ee1455f2134d5b914dd499ccad4377"
+  stream_id: "53441ad2dd9f4e44a76dccab74d0a2ac"
+  errors {
+    error: REPLICATION_MISSING_OP_ID
+    error_detail: "Unable to find expected op id on the producer"
+  }
+}
+```
```
Comment (polarweasel) [inline docs/content/preview/deploy/multi-dc/async-replication.md:154]: We like to split up commands and their output. That was you can copy the command without also copying all the output.

````suggestion
```sh
./bin/yb-admin \
    -master_addresses 127.0.0.1:7000,127.0.0.2:7000,127.0.0.3:7000 \
    get_replication_status
```

```output
statuses {
  table_id: "03ee1455f2134d5b914dd499ccad4377"
  stream_id: "53441ad2dd9f4e44a76dccab74d0a2ac"
  errors {
    error: REPLICATION_MISSING_OP_ID
    error_detail: "Unable to find expected op id on the producer"
  }
}
```
````

File: docs/content/preview/admin/yb-admin.md (hunk 3 above)
Comment (joe-maley) [reply docs/content/preview/admin/yb-admin.md:1851]: We have a mix between `./bin/` and empty prefixes all over this page for `yb-admin`. Do you mind if I go ahead and change them all to be consistent?

File: docs/content/preview/admin/yb-admin.md (hunk 4 above)
Comment (joe-maley) [reply docs/content/preview/admin/yb-admin.md:1864]: Yes, this is correct. It's a pretty-print from proto3, which is used in other yb-admin commands as well.

File: docs/content/preview/admin/yb-admin.md (hunk 3 above)
Comment (ddhodge) [reply docs/content/preview/admin/yb-admin.md:1851]: The convention appears to be not to use the ./bin/ in the syntax, only in the Examples, and it appears to be consistent throughout, so I wouldn't change this...

Comment (polarweasel) [reply docs/content/preview/admin/yb-admin.md:1851]: Thanks for pointing this out! Leave as-is for sure.

**Review by hari90 — APPROVED** [review]

**Pull request merged**